src/js/base/logic.js

```diff
@@ -246,6 +246,26 @@ ripe.Ripe.prototype.hasFrame = function(frame) {
     return true;
 };
 
+/**
+ * Determines if the model currently loaded (in case there's one) has the
+ * provided video name available in spec.
+ *
+ * Notice that this call does not assure that a video showing is possible
+ * it only determines that according to model's spec it should be
+ * possible to get such a video.
+ *
+ * @param {String} frame The name of the video to determine if it is
+ * possible to show that video.
+ * @returns {Boolean} If it's possible to show the video for the
+ * currently loaded model.
+ */
+ripe.Ripe.prototype.hasVideo = function(name) {
+    if (!this.loadedConfig) return true;
+    if (!name) return true;
+
+    return Boolean(this.loadedConfig.videos_m[name]);
+};

```

Comment:
Maybe:

```js
ripe.Ripe.prototype.hasVideo = function(name) {
    if (!name) return true;
    if (!this.loadedConfig || !this.loadedConfig.videos_m) return true;
    return Boolean(this.loadedConfig.videos_m[name]);
};
```

Reply:
I maintained the same structure has the one in `hasFrame`. But I notice that a check is missing, will add